app/cost_service.py

```
@@ -0,0 +1,874 @@
+        
+        try:
+            async with db.get_async_connection() as conn:
+                cursor = conn.cursor()
+                                
+                org_query = """
+                    SELECT id FROM public.organizations 
+                    WHERE id = %s AND deleted_at IS NULL
+                """
+                cursor.execute(org_query, (str(organization_id),))
+                if not cursor.fetchone():
+                    raise Exception(f"Organization with ID {organization_id} not found")
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)